### PR TITLE
feat(provider/oraclebmcs): Implement scale down cluster pipeline stage

### PR DESCRIPTION
**Pull request by owainlewis**
Implement scale down cluster pipeline stage for Oracle BMCS provider.

Once our initial code is merged in, I'll refactor it all to TypeScript.

### CODE REVIEW

File: app/scripts/modules/oracle/pipeline/stages/scaleDownCluster/scaleDownClusterExecutionDetails.controller.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const angular = require('angular');
+
+import {
+  EXECUTION_DETAILS_SECTION_SERVICE
+} from '@spinnaker/core';
```
Comment (anotherchrisberry) [inline app/scripts/modules/oracle/pipeline/stages/scaleDownCluster/scaleDownClusterExecutionDetails.controller.js]: don't really need this

File: app/scripts/modules/oracle/pipeline/stages/scaleDownCluster/scaleDownClusterExecutionDetails.controller.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+module.exports = angular.module('spinnaker.oraclebmcs.pipeline.stage.scaleDownCluster.executionDetails.controller', [
+  EXECUTION_DETAILS_SECTION_SERVICE,
+  require('angular-ui-router').default,
+  require('core/delivery/details/executionDetailsSectionNav.directive.js'),
```
Comment (anotherchrisberry) [inline app/scripts/modules/oracle/pipeline/stages/scaleDownCluster/scaleDownClusterExecutionDetails.controller.js]: remove directive reference

File: app/scripts/modules/oracle/pipeline/stages/scaleDownCluster/scaleDownClusterExecutionDetails.controller.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    initialize();
+
+    let initialized = () => {
+      $scope.detailsSection = $stateParams.details;
+    };
+
+    let initialize = () => executionDetailsSectionService.synchronizeSection($scope.configSections, initialized);
+
+    $scope.$on('$stateChangeSuccess', initialize);
+  });
```
Comment (anotherchrisberry) [inline app/scripts/modules/oracle/pipeline/stages/scaleDownCluster/scaleDownClusterExecutionDetails.controller.js]: remove, use `BaseExecutionDetailsCtrl` in template

**Comment by owainlewis** [comment]
@anotherchrisberry changes implemented. Thanks.

**Comment by anotherchrisberry** [comment]
Just needs a rebase (and for the conflict to be resolved) and we'll get this merged.

**Pull request merged**